tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/clean_mapping_test.py

```diff
@@ -0,0 +1,184 @@
+     8201211115, 1.0, NULL, 0, 0,
+     0, 0, NULL, NULL, '99051',
+     1585845, NULL, NULL, 1585846, NULL, NULL),
+
+    (300, 2, 1585845, '2009-05-15', TIMESTAMP('2009-05-15'),
+     8201211116, 1.0, NULL, 0, 0,
+     0, 0, NULL, NULL, '99054',
+     1585845, NULL, NULL, 1585846, NULL, NULL),
+
+    (400, 3, 1585838, '2009-05-15', TIMESTAMP('2009-05-15'),
+     8201211116, 1.0, NULL, 0, 0,
+     0, 0, NULL, NULL, '99054',
+     1585838, NULL, NULL, 1585839, NULL, NULL)
+""")
+
+
+class CleanMappingExtTablesTest(BaseTest.CleaningRulesTestBase):
+
+    @classmethod
+    @mock.patch('cdr_cleaner.cleaning_rules.clean_mapping.get_mapping_tables')
+    def setUpClass(cls, mock_mapping_tables):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        # Set the test project identifier
+        cls.project_id = os.environ.get(PROJECT_ID)
+
+        # Set the expected test datasets
+        cls.dataset_id = os.environ.get('UNIONED_DATASET_ID')
+        cls.sandbox_id = f'{cls.dataset_id}_sandbox'
+
+        # mock mapping_tables
+        cls.tables = ['_mapping_observation', 'observation_ext']
+        mock_mapping_tables.return_value = cls.tables
+
+        # Instantiate class
+        cls.rule_instance = CleanMappingExtTables(cls.project_id,
+                                                  cls.dataset_id,
+                                                  cls.sandbox_id)
+
+        # Generate sandbox table names
+        sandbox_table_names = cls.rule_instance.get_sandbox_tablenames()
+        for table_name in sandbox_table_names:
+            cls.fq_sandbox_table_names.append(
+                f'{cls.project_id}.{cls.sandbox_id}.{table_name}')
+
+        # Store affected table names
+        affected_tables = cls.rule_instance.affected_tables
+        for table_name in affected_tables:
+            cls.fq_table_names.append(
+                f'{cls.project_id}.{cls.dataset_id}.{table_name}')
+
+        # call super to set up the client, create datasets, and create
+        # empty test tables
+        # NOTE:  does not create empty sandbox tables.
+        super().setUpClass()
+
+    def setUp(self):
+        """
+        Create test table for the rule to run on
+        """
+
+        table_schemas = [
+            self.client.get_table_schema('_mapping_observation'),
+            self.client.get_table_schema('observation_ext'),
+            self.client.get_table_schema(OBSERVATION)
+        ]
+
+        # create tables
+        observation_table = f'{self.project_id}.{self.dataset_id}.{OBSERVATION}'
+        for table, schema in zip(self.fq_table_names + [observation_table],
+                                 table_schemas):
+            table_obj = bigquery.Table(table, schema)
+            self.client.create_table(table_obj, exists_ok=True)
+
```

Comment:
Once we add `OBSERVATION` to `cls.fq_table_names`, we can replace the lines 125-136 to simply `super().setUp()`. The base test class' `setUp` can prepare the tables for you.